plugins/octant/Makefile

```diff
@@ -0,0 +1,13 @@
+SHELL              := /bin/bash
+GO                 ?= go
+BINDIR             ?= $(CURDIR)/bin
```

Comment:
You don't have the long `ANTCTL_BINARY_NAME ?= antctl` so don't need so many spaces.

Reply:
Right, will update this part.